### ISSUE TITLE
Cute Chess seems to be using an old GUI design on Windows 10

### ISSUE THREAD

**Issue by sewbacca** [cutechess/cutechess#675]
## Output of `cutechess-cli --version` / `cutechess --version`

```
cutechess-cli 1.2.0
Using Qt version 5.11.1

Copyright (C) 2008-2020 Ilari Pihlajisto, Arto Jonsson and contributors
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
```

## How did you run Cute Chess exactly?

Started from Windows Start Menu.

## Expected behavior

Look like a Windows 10 Application

## Actual behavior

Looks like a Windows XP Application
(I tested the 1.0 version, and the GUI there looked normal)

## Steps to reproduce the behavior

1. Install Cute Chess via the setup.exe
2. Launch Cute Chess

**Screenshots**

[![Cute-is-not-so-cute.png](https://i.postimg.cc/4NFrTjBQ/Cute-is-not-so-cute.png)](https://postimg.cc/mtCdYpxP)

**Comment by anonymous7002** [comment]
That's... not an error. That's a suggestion.

**Comment by sewbacca** [comment]
> 
> 
> That's... not an error. That's a suggestion.

Why do you think so? If past versions had the newer GUI theme from qt, is reverting back to the older GUI theme not an accident?

**Comment by Oragonon** [comment]
Sewbacca,
The appearance is okay with me. If you don't like the design, try programming your own.

**Comment by Qu4tro** [comment]
Not that I have any stake on the matter, I've only stumbled here semi-randomly.

This clearly seems like a valid issue. If I'm understanding correctly, it had the correct UI on a previous version and it doesn't anymore. There's also no mention of anything related in the release notes.
Being snarky is just not helpful, man.

That said, it seems (also from a quick look) the 1.2.0 release was a long time in the making, so it might not be a trivial change.

Hopefully, you get some feedback from the contributors, to help solve the problem @sewbacca .
Cheers.

**Comment by artoj** [comment]
Can you try 1.3.0-beta: https://github.com/cutechess/cutechess/releases/tag/v1.3.0-beta4

**Comment by sewbacca** [comment]
No, unfortunately it still looks the same as before. Is there any way I can provide logs or something?

**Comment by artoj** [comment]
Thanks. I need to check this on my Windows machine. It might be related to the Windows build using Qt 5.15.

**Comment by artoj** [comment]
I've identified the problem: a missing windows style DLL from the binary. I'll try to fix this for the next 1.3.0 beta.

**Comment by artoj** [comment]
Should be fixed now.
